zebra/zebra_neigh.c

```diff
@@ -118,8 +118,8 @@ static void zebra_neigh_free(struct zebra_neigh_ent *n)
 		/* if rules are still using the neigh mark it as inactive and
 		 * update the dataplane
 		 */
-		if (n->flags & ZEBRA_NEIGH_ENT_ACTIVE) {
-			n->flags &= ~ZEBRA_NEIGH_ENT_ACTIVE;
+		if (CHECK_FLAG(n->flags, ZEBRA_NEIGH_ENT_ACTIVE)) {
```

Comment:
This check is useless at all.

Reply:
Thanks for the review.
Updated as suggested.

Reply:
@ton31337 , can you please review and help to merge the changes if all fine.